python/mxnet/symbol.py

```diff
@@ -1025,6 +1025,36 @@ def grad(self, wrt):
         return Symbol(handle)
     # pylint: enable= no-member
 
+    def eval(self, args=None, ctx=cpu()):
+        if args == None:
```

Comment:
I'm not sure your suggestion makes sense:

```
How about eval(ctx=cpu(), **kwargs)?
eval(x=..., y=...) is more handy than eval({'x': ..., 'y': ...})
```

1) In this case ctx is a keyword argument...
2) While it's a little handier to not have to explicitly pass in the dictionary, the dictionary style is closer to bind. I'd rather not have such completely different interfaces.